apps/hash-api/turbo.json

```diff
@@ -4,7 +4,7 @@
     "codegen": {
       // Outputs of GraphQL generation
       "outputs": ["./src/**/*.gen.*"],
-      "dependsOn": ["^build"]
+      "dependsOn": ["^codegen"]
```

Comment:
This change may be incorrect, but I couldn't tell why this was depending on `build` for its dependencies